README.md

```diff
@@ -1,66 +1,79 @@
-                     --own-pub-key-hash PUB_KEY_HASH
+                     [--beneficiaries-file FILENAME] [--use-pub-key-hashes]
+                     (--own-address ADDRESS | --own-pub-key-hash PUB_KEY_HASH)
                      [--signing-key-file FILENAME]
-                     --beneficiaries-per-tx NATURAL 
-                     [--dry-run]
-                     --min-lovelaces NATURAL
-                     --fees NATURAL
+                     [--asset-class CURRENCY_SYMBOL.TOKEN_NAME]
+                     [--drop-amount NATURAL] --beneficiaries-per-tx NATURAL
+                     [--dry-run] --min-lovelaces NATURAL --fees NATURAL
+  CLI tool to simplify sending native tokens to multiple users
 ```
 
 - `mainnet` OR `testnet-magic NATURAL`: This should match whichever node you have set up
 - `protocol-params-file ARG`: This will default to `./config/protocol.json`
-- `asset-class`: The asset class of the token to be distributed
 - `beneficiaries-file`: The file containig the above beneficiaries format. This will default to `./config/beneficiaries`
 - `use-pub-key-hashes`: Sets the beneficiary file to accept PubKeyHashes over addresses
 - `own-pub-key-hash`: PubKeyHash of the address holding the tokens to be distributed
 - `signing-key-file`: Signing key file of the above PubKeyHash. This will default to `./config/server.skey`
+- `asset-class`: Token asset class (overrides beneficiaries file config)
+- `drop-amount`: Amount of tokens to send to each beneficiary (overrides beneficiaries file config)
 - `beneficiaryPerTx`: This controls how many transaction outputs we batch together. In case the tranaction exceeds the size limit, try to change this value
 - `dryRun`: Builds transactions without actually submitting them on chain
 - `min-lovelaces`: Minimun lovelace amount for each utxo (change this it you get a Minimum required UTxO error)
 - `fees`: Transaction fees, this is only used for coin selection, the actual fee is calculated by the cardano-cli
 
 ### Beneficiaries format
+
 This file is structured as one receipient per line, with the following format:  
-`address amount`  
-If the `use-pub-key-hashes` flag is set, the addresses become pub key hases
+`address amount currencySymbol.tokenName`
+
+- If the `use-pub-key-hashes` flag is set, the addresses become pub key hashes
+- If the `asset-class` and/or `drop-amoun` option is present, the values specified on the cli take precedence over the ones in the file (in this case these fields are optional, however the )
+
 #### Examples
+
 Without `use-pub-key-hashes`:
+
 ```
-addr_test1vpzm7yazemjns5plryg9yq9lkv9xzr432e88jsdqprty4fqhcw9d7 1001
-addr_test1vqsk6udkq2a552prwtmpmfs8yqyluxlvgx6zy20tqjsglyctfkjrg 1002
-addr_test1vz0vpcef37gsanrj8mtta9hkhfd3ja5ekq7mdjgays7wzlcwzmvf6 1003
-addr_test1vq85t2h3k22emdh9l72dhv0cywlj2a5qc0rj8tpdf8uh23st77ahh 1004
+addr_test1vpzm7yazemjns5plryg9yq9lkv9xzr432e88jsdqprty4fqhcw9d7 1001 1d6445ddeda578117f393848e685128f1e78ad0c4e48129c5964dc2e.testToken
```

Comment:
Perhaps some more examples of the various ways the Beneficiary file changes when using command line args